### ISSUE TITLE
Auth.signInWithWebUI on iOS 14.5 causes app sigterm

### ISSUE THREAD

**Issue by martincalvert** [aws-amplify/amplify-flutter#566]
**Describe the bug**
After updating the iOS simulator to 14.5, the Auth.signInWithWebUI call causes the app to crash with no exceptions or logs, but the debug console says "Lost connection to device." and the "Exited (sigterm)". The thing I noticed is the dialog to ask about opening a URL is not showing up in 14.5 but shows up in 14.4

**To Reproduce**
1. Setup a login flow using 3rd party providers and the hosted UI
2. Attempt to sign in with apple or google
3. Observe sigterm

**Expected behavior**
After clicking on sign in with apple, it is expected for a dialog to be presented to request permission to open the cognito hosted UI URL, after which you should be able to sign in with the selected provider.

**Screenshots**
N/A

**Platform**
Amplify Flutter current supports iOS and Android.  This issue is reproducible in (check all that apply):
[] Android
[x] iOS

<details>

  <summary>Output of <code>flutter doctor -v</code></summary>
  
  ```[flutter] flutter doctor -v
[✓] Flutter (Channel stable, 2.0.3, on macOS 11.0.1 20B50 darwin-x64, locale en-US)
    • Flutter version 2.0.3 at /Users/martincalvert/bin/flutter
    • Framework revision 4d7946a68d (7 weeks ago), 2021-03-18 17:24:33 -0700
    • Engine revision 3459eb2436
    • Dart version 2.12.2

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Users/martincalvert/Library/Android/sdk
    • Platform android-30, build-tools 30.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.5, Build version 12E262
    • CocoaPods version 1.10.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[✓] VS Code (version 1.56.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.21.0

[✓] Connected device (4 available)
    • iPhone 12 (mobile)         • 29CBD676-C363-4103-A272-B4644042F9D0 • ios            • com.apple.CoreSimulator.SimRuntime.iOS-14-4 (simulator)
    • iPhone 12 mini (mobile)    • 013A3FFF-5150-496A-8AE3-FCDC8D8C94B8 • ios            • com.apple.CoreSimulator.SimRuntime.iOS-14-5 (simulator)
    • iPhone 12 Pro Max (mobile) • 21299780-A19E-4ACF-A295-83BF42B6DE33 • ios            • com.apple.CoreSimulator.SimRuntime.iOS-14-5 (simulator)
    • Chrome (web)               • chrome                               • web-javascript • Google Chrome 90.0.4430.93

• No issues found!```

</details>

**Smartphone (please complete the following information):**
 - Device: iPhone *
 - OS: iOS 14.5

**Additional context**
Running the debugger and stepping into that method instant crashes as well.

**Comment by Jordan-Nelson** [comment]
Hi @martincalvert - I believe this is caused by a bug in the iOS SDK which was fixed a while back (see [relevant issue](https://github.com/aws-amplify/amplify-ios/issues/1086)). Your project likely still has the older version of the iOS SDK that does not have the fix. You can verify this by looking at what version of AWSAuthCore/AWSCore is listed in  `ios/Podfile.lock`. The fix was released in 2.23.1. You should be able to update to the latest version by running `pod update` from the ios dir, which should resolve this issue.

CC: @haverchuck I think #540 would allow this to be resolved just by updating to the latest version of amplify_flutter.

**Comment by cto-leaps** [comment]
Hey @Jordan-Nelson 
I'm on 14.5 and my AWSMobileClient is at version 2.24.1.
However, I have the same issue....
Any clue???

**Comment by Jordan-Nelson** [comment]
Hello @cto-leaps - Is this working for you on iOS 14.4?

If it is not working on iOS 14.4 either, can you confirm you have followed the docs on signInWithWebUI, in particular [the section on iOS setup](https://docs.amplify.aws/lib/auth/signin_web_ui/q/platform/flutter#ios-platform-setup). 

If it is working as expected on iOS 14.4, can you let me know what version of `Amplify` and `AmplifyPlugins/AWSCognitoAuthPlugin` are in your Podfile.lock?

**Comment by Jordan-Nelson** [comment]
For anyone else experiencing this issue, the current recommendation is to run `pod update Amplify AWSPluginsCore AmplifyPlugins` from the iOS directory of your project. This should update all amplify related pods to the latest version. You can confirm they have updated by looking at your podfile.lock before and after running it.

For a longer term fix, we are looking into adding a min version of the amplify pods in amplify flutter so that this can be addressed by upgrading to the latest version of amplify flutter. That has an open PR - https://github.com/aws-amplify/amplify-flutter/pull/589

**Comment by haverchuck** [comment]
@martincalvert I believe this should be resolved by using amplify-flutter 0.1.6 (and possibly updating your pods).  Please let us know.  Will leave this issue open for comment.

**Comment by martincalvert** [comment]
Sorry for the delay. That resolved it. Thanks!
